tests/migrations/test_commands.py

```diff
@@ -338,6 +339,127 @@ def test_showmigrations_plan_squashed(self):
             out.getvalue().lower()
         )
 
+    @override_settings(INSTALLED_APPS=[
+        "migrations.migrations_test_apps.mutate_state_b",
+        "migrations.migrations_test_apps.alter_fk.author_app",
+        "migrations.migrations_test_apps.alter_fk.book_app",
+    ])
+    def test_showmigrations_plan_single_app_label(self):
+        """
+        Tests `--plan app_label` output of `showmigrations`.
+        """
+        # Single app with no dependencies on other apps.
+        out = six.StringIO()
+        call_command("showmigrations", "mutate_state_b", format='plan', stdout=out)
```

Comment:
You could use single quotes in all strings for consistency (don't worry about existing tests).